pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -70,6 +74,7 @@ def resume_start(self, checkpoint_path: Optional[_PATH] = None) -> None:
         self.resume_checkpoint_path = self.hpc_resume_path or checkpoint_path
         checkpoint_path = self.resume_checkpoint_path
         if not checkpoint_path:
+            log.info("`checkpoint_path` not specified. Skipping checkpoint loading.")
```

Comment:
should this be verbose?
```suggestion
            log.verbose("`checkpoint_path` not specified. Skipping checkpoint loading.")
```